projects/ion/src/lib/input/input.component.html

```diff
@@ -47,7 +48,16 @@
       size="16"
     >
     </ion-icon>
+    <button
+      *ngIf="isClearButtonVisible()"
+      data-testid="clear-button"
+      class="clearButton"
+      (click)="clearInput(); input.value = ''"
```

Comment:
I understand why you do this attribution here (to check input value on tests, right?), but I don't think this is a nice approach, since you're already doing this inside the function.
You can keep this for now, but I think we need to find out why input value is not updating to check it on jest properly.